packages/threedimensional/src/scenes/World/index.ts

```diff
@@ -0,0 +1,196 @@
+//Commonjs import of three
+import { normalizeValue } from '../../utils';
+import * as THREE from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+
+class World {
+  renderer!: THREE.WebGLRenderer;
```

Comment:
why do you need the `!` in props declaration? (applies to all others props)

Reply:
If you run the typescript compiler in strict mode and define those class fields in a method outside the constructor. Typescript will complain about the class field having the possibility of being undefined since it's not being declared in the constructor. Since I'm not running it in strict mode anymore and composer isn't running in strict mode either I removed them 👍🏼